addons/TBMod_main/Cfg3DEN.hpp

```diff
@@ -157,6 +157,24 @@ class Cfg3DEN
                     // };
                 };
             };
+            class TBMod_main_schauspieler
```

Comment:
Ist dafür eine ganze neue Kategorie notwenig?

Ggf an `class TB_arsenal` ranhängen:
https://github.com/shukari/TBMod/blob/a87902c6f17daa2ed495fcf27e7147beaefe59d1/addons/TBMod_arsenal/configs/Cfg3DEN.hpp#L85

oder wir nennen `TB KI-Aktionen` einfach in TB Aktionen bzw TB Eigenschaften um, da kann dann auch das Arsenal rein... brauchen keine 100 Untermenüs für eine Sache, nicht dein Fehler generelle Designsache ;D